commit 839fad996bf0eba85c0c6aa67ba8d89d051b6749
Author: nohee ko <noheek@broadcom.com>
Date:   Sat Oct 9 10:41:05 2010 -0700

    staging: brcm80211: remove unnecessary cflag, LINUX
    
    remove unnecessary cflag, LINUX
    
    Signed-off-by: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmfmac/Makefile
+++ b/drivers/staging/brcm80211/brcmfmac/Makefile
@@ -28,7 +28,6 @@ ccflags-y :=			\
 	-DDHD_SCHED		\
 	-DDHD_SDALIGN=64	\
 	-DEMBEDDED_PLATFORM	\
-	-DLINUX			\
 	-DMAX_HDR_READ=64	\
 	-DMMC_SDIO_ABORT	\
 	-DPKT_FILTER_SUPPORT	\
--- a/drivers/staging/brcm80211/brcmfmac/dhd.h
+++ b/drivers/staging/brcm80211/brcmfmac/dhd.h
@@ -21,7 +21,6 @@
 #ifndef _dhd_h_
 #define _dhd_h_
 
-#if defined(LINUX)
 #include <linux/sched.h>
 #include <linux/init.h>
 #include <linux/kernel.h>
@@ -38,14 +37,6 @@
 #include <linux/wakelock.h>
 #endif			/* defined (CONFIG_HAS_WAKELOCK) */
 /* The kernel threading is sdio-specific */
-#else				/* LINUX */
-#define ENOMEM		1
-#define EFAULT      2
-#define EINVAL		3
-#define EIO			4
-#define ETIMEDOUT	5
-#define ERESTARTSYS 6
-#endif				/* LINUX */
 
 #include <wlioctl.h>
 
